CONTRIBUTING.md

```diff
@@ -11,9 +11,15 @@ In the command line:
 
 1. Clone the repo by typing `git clone https://github.com/YOUR-GITHUB-USERNAME/Developer_Quiz_Site.git`
 2. Then type `cd Developer_Quiz_Site` to go into the project's directory.
-3. Install dependencies with `npm install`
-4. Run the project with `npm start`
-5. Have fun 🚀
+    - Option A: NPM
+      1. Ensure Node.js and NPM are installed on machine
+      2. Install dependecies with `npm install`
```

Comment:
Users will still have to install dependencies to be able to use the prettier pre commit hook. Also they will still have to ensure that Node and NPM are installed. 

These two steps should be moved to the general setup steps.